code/game/machinery/cloning.dm

```diff
@@ -249,22 +249,23 @@
 	if(exchange_parts(user, W))
 		return
 
-	default_deconstruction_crowbar(W)
+	if(default_deconstruction_crowbar(W))
+		return
 
-	if (istype(W, /obj/item/weapon/card/id)||istype(W, /obj/item/device/pda))
-		if (!src.check_access(W))
+	if (W.GetID())

```

Comment:
GetID grabs id cards and PDAs (and maybe more items that contains access in the future)